src/components/Common/UserSelector.tsx

```diff
@@ -55,8 +55,8 @@ export default function UserSelector({
   const users = data?.results || [];
 
   return (
-    <Popover open={open} onOpenChange={setOpen}>
-      <PopoverTrigger asChild>
+    <Popover open={open} onOpenChange={setOpen} modal={true}>
+      <PopoverTrigger asChild className="w-full">
```

Comment:
I'd recommend passing it as a prop here, since this maybe used in multiple other places where w-full maybe undesired.

Reply:
> I'd recommend passing it as a prop here, since this maybe used in multiple other places where w-full maybe undesired.

Needed it in both places we are currently using it, but yea could be the case in the future so added as a prop 👍